### PR TITLE
Fix: Make version changeable from the environment vars

### LINKED ISSUE #4648: Facebook authentication issue on v13
The issue occurs while authenticating. The link that presents v12. but the Facebook application is in v13.
unknown

### PR DESCRIPTION
**Pull request by tejaswinichile**
# Pull Request Template

## Description

Facebook updates the version and self-hosted clients might be creating Facebook apps with different versions so instead of making changes in every line of code, we will be introducing the FB_API_VERSION env variable, so it will be configurable for non-tech people too.

Fixes #4648 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

Tested on staging.

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by tejaswinichile** [comment]
https://www.loom.com/share/7ee74a147cd843648405e9cdb2526b43


Testing is done on staging.

**Comment by pranavrajs** [comment]
@tejaswinichile Should this be documented on the docs or is this PR just to add an ability to configure it via env var only when it is required?

**Comment by tejaswinichile** [comment]
@pranavrajs I am making changes in the docs too. This change is required for all the future self-hosted versions.

**Comment by sojan-official** [comment]
lets bump up the API version with out introducing a new env variable

**Review by sojan-official — APPROVED** [review]

File: app/javascript/dashboard/helper/scriptGenerator.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ export const createMessengerScript = pageId => `
     FB.init({
       appId: "${window.chatwootConfig.fbAppId}",
       xfbml: true,
-      version: "v4.0"
+      version: "${window.chatwootConfig.fbApiVersion}"
```
Comment (pranavrajs) [inline app/javascript/dashboard/helper/scriptGenerator.js:7]: @tejaswinichile This change should not work as it renders as

`${window.chatwootConfig.fbApiVersion}` not `v12.0`.

The change should have been 
```
`${window.chatwootConfig.fbApiVersion}`
```

Comment (pranavrajs) [reply app/javascript/dashboard/helper/scriptGenerator.js:7]: Figured that the code is not used, hence removing it.

https://github.com/chatwoot/chatwoot/pull/4671

**Pull request merged**